docs/fr/docs/advanced/additional-responses.md

```diff
@@ -0,0 +1,240 @@
+# Réponses supplémentaires dans OpenAPI
+
+!!! Attention
+    Ceci concerne un sujet plutôt avancé.
+
+    Si vous débutez avec **FastAPI**, vous n'en aurez peut-être pas besoin.
+
+Vous pouvez déclarer des réponses supplémentaires, avec des codes HTTP, des types de médias, des descriptions, etc.
+
+Ces réponses supplémentaires seront incluses dans le schéma OpenAPI, elles apparaîtront donc également dans la documentation de l'API.
+
+Mais pour ces réponses supplémentaires, vous devez vous assurer de renvoyer directement une `Response` comme `JSONResponse`, avec votre code HTTP et votre contenu.
+
+## Réponse supplémentaire avec `model`
+
+Vous pouvez ajouter à votre décorateur de *paramètre de chemin* un paramètre `responses`.
+
+Il prend comme valeur un `dict` dont les clés sont des codes HTTP pour chaque réponse, comme `200`, et la valeur de ces clés sont d'autres `dict` avec des informations pour chacun d'eux.
+
+Chacun de ces `dict` de réponse peut avoir une clé `model`, contenant un modèle Pydantic, tout comme `response_model`.
+
+**FastAPI** prendra ce modèle, générera son schéma JSON et l'inclura au bon endroit dans OpenAPI.
+
+Par exemple, pour déclarer une autre réponse avec un code HTTP `404` et un modèle Pydantic `Message`, vous pouvez écrire :
+
+```Python hl_lines="18 22"
+{!../../../docs_src/additional_responses/tutorial001.py!}
+```
+
+!!! Remarque
+    Gardez à l'esprit que vous devez renvoyer directement `JSONResponse`.
+
+!!! Info
+    La clé `model` ne fait pas partie d'OpenAPI.
+
+    **FastAPI** prendra le modèle Pydantic à partir de là, générera le `JSON Schema` et le placera au bon endroit.
+
+    Le bon endroit est :
+
+    * Dans la clé `content`, qui a pour valeur un autre objet JSON (`dict`) qui contient :
+        * Une clé avec le type de support, par ex. `application/json`, qui contient comme valeur un autre objet JSON, qui contient :
+            * Une clé `schema`, qui a pour valeur le schéma JSON du modèle, voici le bon endroit.
+                * **FastAPI** ajoute ici une référence aux schémas JSON globaux à un autre endroit de votre OpenAPI au lieu de l'inclure directement. De cette façon, d'autres applications et clients peuvent utiliser ces schémas JSON directement, fournir de meilleurs outils de génération de code, etc.
+
+Les réponses générées au format OpenAPI pour cette *opération de chemin* seront :
+
+```JSON hl_lines="3-12"
+{
+    "responses": {
+        "404": {
+            "description": "Réponse supplémentaire",
+            "content": {
+                "application/json": {
+                    "schema": {
+                        "$ref": "#/composants/schémas/Message"
```

Comment:
Why not about the description but please do not translate the ref, we need to keep consistent with the example (`{!../../../docs_src/additional_responses/tutorial001.py!}`, [here ](https://fastapi.tiangolo.com/fr/advanced/additional-responses/#additional-response-with-model))
```suggestion
                        "$ref": "#/components/schemas/Message"
```

this suggestion applicable for the following refs as well